src/main/java/com/codeborne/selenide/CollectionCondition.java

```diff
@@ -173,6 +174,45 @@ public static CollectionCondition itemWithText(String expectedText) {
     return new ItemWithText(expectedText);
   }
 
+  /**
+   * Check that the given collection contains elements with given texts.
+   * <p></p>
+   * Examples:
+   * <pre code='java'>
+   * // collection 1: [Tom, Dick, Harry]
+   * $$("collection 1 locator").should(containTexts("Tom", "Dick", "Harry); // success
```

Comment:
(minor) missed trailing quotation mark `"Harry);` here and next

Reply:
Thank you. 
Fixed all 6 occurrences.